src/qml/pages/onboarding/OnboardingStorageLocation.qml

```diff
@@ -22,12 +22,9 @@ InformationPage {
     description: qsTr("Where do you want to store the downloaded block data?")
     descriptionMargin: 20
     detailActive: true
-    detailItem: ColumnLayout {
-        spacing: 0
-        StorageLocations {
-            Layout.maximumWidth: 450
-            Layout.alignment: Qt.AlignCenter
-        }
+    detailItem: StorageLocations {
+        width: Math.min(parent.width, 450)
```

Comment:
there's an intricate issue with this where this will start off fine being a maximum of 450 px width, but if you resize the window to be a smaller width, then resize to be larger, the width of the storage location OptionButtons will become larger than 450px. I assume this would be rare to run into, but the code has an issue (albeit rare) that isn't present with the code on master:
![settings-to-long](https://user-images.githubusercontent.com/23396902/207944891-88d4d5bb-a2a1-4136-b677-0593d9b2eb07.png)

OnboardingStorageAmount.qml is immune to this because it keeps the nested ColumnLayout method since it is actually appropriate there.

Reply:
Yeah i need to rework this.  I was thinking about these pieces wrong.  There is a conflict with the InformationPage column widths.

Reply:
this issue is fixed now as of https://github.com/bitcoin-core/gui-qml/commit/f9f46e77ad9a7aedd5dcca3225fefcec218422c9, thanks!